src/compiler/diagnosticMessages.json

```diff
@@ -3519,6 +3519,14 @@
         "category": "Error",
         "code": 7036
     },
+    "Create namespace objects for ECMAScript imports which are neither callable nor constructable, implies `allowSyntheticDefaultImports`.": {
+        "category": "Message",
+        "code": 7037
+    },
+    "Import is called or constructed, which is not valid ES2015 module usage, and will fail at runtime.": {

```

Comment:
A namespace-style import cannot be called or constructed, and will cause a failure at runtime.

Reply:
Done